documentation/src/main/asciidoc/search-dsl.asciidoc

```diff
@@ -254,3 +254,12 @@ or skip analysis completely with `.skipAnalysis()`.
 
 See <<search-dsl-predicate>> for more information about how to create predicates
 and about the available options.
+
+=== Incompatible decimal scale
```

Comment:
This really boils down to a different encoding, which has its own section above. I'll update this in another commit.